test/verify/machineslib.py

```diff
@@ -1294,8 +1295,9 @@ def fill(self):
             b.set_input_text("#memory-size", str(self.memory_size))
             b.select_from_dropdown("#memory-size-unit-select", self.memory_size_unit)
 
-            b.set_input_text("#storage-size", str(self.storage_size))
-            b.select_from_dropdown("#storage-size-unit-select", self.storage_size_unit)
+            if self.storage_size is not None:
+                b.set_input_text("#storage-size", str(self.storage_size))
+                b.select_from_dropdown("#storage-size-unit-select", self.storage_size_unit)
```

Comment:
Suggestion: If it's `None`, check that `#storage-size` is not present. This makes tests fail more meaningfully if the parameter is accidentally forgotten.